### DISCUSSION TITLE
Question: Is-it possible to load SVG by symbol ?

### DISCUSSION THREAD

**Discussion by ChristopheI** [wieslawsoltes/Svg.Skia#213] [Ideas]
Sorry if this question is obvious but I didn't find anything about this digging in the code or in the previous issues:

I have a SVG file with several symbol nodes. I would like to load image by symbol.

Example of SVG:
```
<svg xmlns="http://www.w3.org/2000/svg">
    <symbol id="about--filled" viewBox="0 0 24 24">
        <path d="M12 2a10 10 0 1010 10A10 10 0 0012 2zm1 15h-2v-6h2zm0-8h-2V7h2z"/>
    </symbol>
    <symbol id="active-speaker-hide" viewBox="0 0 24 24">
        <path d="M21.25 3.29H7.08l2 2h12.17v12h-.16l1.47 1.48a2 2 0 00.69-1.49v-12a2 2 0 00-2-1.99zm-1.02 15.98l-2-2-12-12-2-2-2.07-2.06L.75 2.62 1.93 3.8a2 2 0 00-.68 1.49v12a2 2 0 002 2H17.4l3.52 3.52 1.42-1.41-2.11-2.11zm-10.81-8H7.25v-2h.17zm-6.17 6v-12h.15l2 2h-.15v6h6.17l4 4z"/>
        <path d="M11.08 7.31l2.17 2.17-.02-2.19-2.15.02z"/>
    </symbol>
</svg>
```

I would like to load "about--filled" or "active-speaker-hide".
Thx

**Comment by wieslawsoltes** [comment]
I'm not sure there is an easy way, probably you can load model and access symbol drawables and render them separately, but not something simple.

https://github.com/wieslawsoltes/Svg.Skia/blob/dd0be69fef36a13d8bbe82819c3639332564c17d/src/Svg.Skia/SKSvg.Model.cs#L94

https://github.com/wieslawsoltes/Svg.Skia/blob/dd0be69fef36a13d8bbe82819c3639332564c17d/src/Svg.Model/Drawables/Elements/SymbolDrawable.cs#L7

**Comment by ChristopheI** [comment]
What I did:
- load the XML/SVG document as XmlDocument
- find a "symbol" node with the correct id attribute value
- get the XML string from the node found
- in this string, rename "symbol" node by "svg" node
- use this string to load a SVG as usual
